src/vs/editor/contrib/codeAction/browser/codeActionUi.ts

```diff
@@ -55,6 +55,24 @@ export class CodeActionUi extends Disposable {
 	override dispose() {
 		this.#disposed = true;
 		super.dispose();
+
+	}
+
+	public hideCodeActionWidget() {
+		this._codeActionWidget.getValue().hideCodeActionWidget();
```

Comment:
Same here about `getValue`. Check that the value exists before trying to access it